modules/seqexec/server/src/main/scala/seqexec/server/gmos/GmosKeywordReader.scala

```diff
@@ -26,18 +32,66 @@ final case class GmosObsKeywordsReader[F[_]: Sync](config: Config) {
           .getOrElse(YesNoType.NO))
       .map(_.toBoolean)
       .safeValOrDefault
+
+  def nodMode: F[String] = "STANDARD".pure[F]
+
+  def nodPix: F[Int] = Gmos.nodAndShuffle(config).map(_.rows).explainExtractError
+
+  def nodCount: F[Int] = Gmos.nodAndShuffle(config).map(_.cycles).explainExtractError
+
+  def nodAxOff: F[Double] = Gmos.nodAndShuffle(config).explainExtractError
```

Comment:
These four methods are identical other than the stage and angle selector. I think you forgot to change the error message in the last two when you copy/pasted. Can we factor out one method and call it with different arguments?